### PR TITLE
Updating import instructions for scipy, sklearn - issue #848

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR is addressing the issue #848, which notes that you have import sklearn before nest, as well as scipy. This is not a fix but just a documentation change to make people aware of it. I added a comment to the PyNEST tutorial regarding the import order of both scipy and sklearn. Additionally, I added a line to indicate the import order (wrt scipy) in the CampbellSiegert example, since it was the only example that used scipy that did not explicitly mention the order needed.  
There are also a couple of minor changes to the PyNEST tutorial, unrelated to the import issue but brought to my attention as mistakes in the docs that I hope would be ok to add into this PR.

### CODE REVIEW

File: pynest/examples/CampbellSiegert.py [hunk 1]
```diff
@@ -34,6 +34,11 @@
 Sven Schrader, Nov 2008, Siegert implementation by Tom Tetzlaff
 '''
 
+'''
+First, we import all necessary modules for simulation and analysis. Scipy
```
Comment (Silmathoron) [inline pynest/examples/CampbellSiegert.py]: On my computer, putting the two scipy imports after nest does not lead to any kind of trouble, can you post what the error message you get is?
(python 3.6.4, scipy 0.19)

File: extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md [hunk 2]
```diff
@@ -66,12 +66,22 @@ Python interpreter.
 
     import nest
 
+
+It should be noted, however, that certain external packages must be imported *before* importing nest. 
+These include `sklearn` ([scikit-learn](http://scikit-learn.org/stable/index.html)) and `scipy` ([SciPy](https://www.scipy.org/)):
+
+    from sklearn.svm import LinearSVC
```
Comment (Silmathoron) [inline extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md]: I can confirm the sklearn bug on Python 3.6.4 and SKL 0.19.1, but not with scipy.
Also importing only ``sklearn`` and not the specific submodule does not solve the problem.

File: extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md [hunk 3]
```diff
@@ -66,12 +66,22 @@ Python interpreter.
...
+    import nest
+
+
 As with every other module for Python, the available functions can be prompted
 for.
 
     dir(nest)
 
-One such command is `nest.Models?`, which will return a list of all the
+One such command is `nest.Models?`, which will return a list of all the
```
Comment (jougs) [inline extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md]: Please remove trailing whitespace

File: pynest/examples/CampbellSiegert.py (hunk 1 above)
Comment (jougs) [reply pynest/examples/CampbellSiegert.py]: The original issue we experienced was https://github.com/numpy/numpy/issues/2521, apparently fixed in https://github.com/numpy/numpy/commit/a226daa75816577ac9c734133e600e5eebd9aaa9.

As the suggested import order does not matter in most cases and prevents a crash in the best case, I like the proposed addition although most people will not be hit by the long-fixed issue anymore.

File: extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md [hunk 4]
```diff
@@ -248,8 +258,7 @@ Now we are ready to display the data in a figure. To this end, we make use of
     pylab.figure(1)
     pylab.plot(ts, Vms)
 
-The second line opens a figure (with the number 1), the third line clears the
-window and the fourth line actually produces the plot. You can’t see it yet
+The second line opens a figure (with the number 1), and the third line actually produces the plot. You can’t see it yet
```
Comment (jougs) [inline extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md]: Please break the like in a similar fashion as before.

File: extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md (hunk 2 above)
Comment (jougs) [reply extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md]: @Silmathoron thanks for double-checking. In this case I agree with the suggested changes.

**Comment by heplesser** [comment]
@jougs @Silmathoron Are you happy with the PR now?

**Pull request merged**